src/features/DisplayControls/DrugChart/components/TimeCell.jsx

```diff
@@ -72,7 +80,11 @@ export default function TimeCell(props) {
       <div
         data-testid="right-icon"
         className={
-          doHighlightCell && highlightedCell === "right" ? "highligtedCell" : ""
+          doHighlightCell && highlightedCell === "right"
+            ? "highlightedCell"
+            : isBlank
+            ? "blank"
+            : ""
```

Comment:
I think we don't need a condition blank class here just give the drugchart parent background color as white